File: Framework/Java/framework/framework/smartkernel/framework/SmartConsole.java
/*
 * 项目：Smartkernel.Framework
 * 作者：曹艳白
 * 邮箱：caoyanbai@139.com
 */
package smartkernel.framework;

import java.util.Scanner;

/**
 * Console
 * 
 */
public class SmartConsole {
	/**
	 * 输出消息
	 * 
	 * @param message
	 *            ，消息
	 */
	public static void write(Object message) {
		System.out.print(message);
	}

	/**
	 * 输出消息
	 * 
	 * @param message
	 *            ，消息
	 */
	public static void writeLine(Object message) {
		System.out.println(message);
	}

	/**
	 * 输入消息
	 * 
	 * @return，消息
	 */
	public static String read() {
		try (Scanner scanner = new Scanner(System.in)) {
			return scanner.next();
		}
	}

	/**
	 * 输入消息
	 * 
	 * @return，消息
	 */
	public static String readLine() {
		try (Scanner scanner = new Scanner(System.in)) {
			return scanner.nextLine();
		}
	}
}

File: Framework/Php/framework/framework/smartconsole.php
<?php

/*
 * 项目：Smartkernel.Framework
 * 作者：曹艳白
 * 邮箱：caoyanbai@139.com
 */

namespace Smartkernel\Framework {

    use Smartkernel\Framework\Computer\SmartComputer;

/*
     * 输出
     */

    class SmartConsole {
        /*
         * 输出
         * 
         * @param object $input，输入
         * 
         */

        public static function write($input) {
            echo strval($input);
        }

        /*
         * 换行输出
         * 
         * @param object $input，输入
         * 
         */

        public static function writeLine($input) {
            echo strval($input) . SmartComputer::LineSeparator;
        }

    }

}